### PR TITLE
Runtime release notes draft

### PR DESCRIPTION
**Pull request by lzdanski**
See https://astronomer.slack.com/archives/C020PVBJPKK/p1685105449391679

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -26,6 +26,23 @@ To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For g
 
 ## Astro Runtime 8.2.0
 
+- Release date: May 26, 2023
+- Airflow version: 2.6.1
+
+### Additional Improvements
+
+- Upgraded astronomer-providers to 1.16.0
+- Upgraded openlineage-airflow to 0.26.0
```
Comment (ephraimbuddy) [inline astro/runtime-release-notes.md:35]: We should add a link to the provider changelog. See https://github.com/astronomer/docs/pull/2297/files

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -26,6 +26,23 @@ To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For g
...
+- Blocked the ability to pause the Monitoring DAG with the Airflow API. The Monitoring DAG is used by Astronomer to operate your Deployments and should not be paused.
+- Adds Datadog as a provider. See the GitHub Provider Page on the [Astronomer Registry](https://registry.astronomer.io/providers/apache-airflow-providers-datadog/versions/3.3.0).
+
+### Bug fixes
+
+- Fixed a bug to ensure that `min_backoff` in the base sensor is at least `1`.
+- Updated error messaging.
```
Comment (ephraimbuddy) [inline astro/runtime-release-notes.md:42]: Links to these backported changes too https://github.com/astronomer/docs/pull/2067/files

File: astro/runtime-release-notes.md [hunk 3]
```diff
@@ -26,6 +26,23 @@ To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For g
...
 
+- Release date: May 26, 2023
+- Airflow version: 2.6.1
+
+### Additional Improvements
+
+- Upgraded `astronomer-providers` to 1.16.0. See the [`astronomer-providers` changelog](https://github.com/astronomer/astronomer-providers/blob/main/CHANGELOG.rst#1160-2023-05-19) for a complete list of changes. 
+- Upgraded openlineage-airflow to 0.26.0. See the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/releases/tag/0.26.0) for a complete list of changes. 
+- Blocked the ability to pause the Monitoring DAG with the Airflow API. The Monitoring DAG is used by Astronomer to operate your Deployments and should not be paused.
+- Adds Datadog as a provider. See the GitHub Provider Page on the [Astronomer Registry](https://registry.astronomer.io/providers/apache-airflow-providers-datadog/versions/3.3.0).
```
Comment (jedcunningham) [inline astro/runtime-release-notes.md:42]: I feel like "Adds the Datadog provider", or similar, is better (Datadog itself isn't a provider).

File: astro/runtime-release-notes.md (hunk 2 above)
Comment (lzdanski) [reply astro/runtime-release-notes.md]: Backported changes? I'm a little confused, I've added a link to the Airflow changelog, but is there something else that should go here?

File: astro/runtime-release-notes.md (hunk 3 above)
Comment (jedcunningham) [reply astro/runtime-release-notes.md:42]: Did this get missed?

Comment (lzdanski) [reply astro/runtime-release-notes.md]: it sure did! fixed now.

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**